src/ui/Button.cpp

```diff
@@ -117,7 +117,8 @@ void Button::draw(SDL_Rect _size, SDL_Rect _actualSize) {
 				scaledPos.y = pos.y * (float)yres / (float)Frame::virtualScreenY;
 				scaledPos.w = pos.w * (float)xres / (float)Frame::virtualScreenX;
 				scaledPos.h = pos.h * (float)yres / (float)Frame::virtualScreenY;
-				_text->drawColor(section, scaledPos, textColor);
+				//TODO: If we want to go with this method, we have to recalculate the point size here...
+				_font->drawTextColor(text, scaledPos.x, scaledPos.y, textColor);
```

Comment:
If we don't scale a prerendered buffer, we need to modify the code at these points, whoever calls drawTextColor() needs to pass in the point size for their desired scaling...

Reply:
Not sure on the exact implementation, but off the top of my head I might request to render my string at pointsize 10, take a measurement, then compute the maximum pointsize I can use to keep within the desired scaled dimensions.
Alternatively, we can just choose a standard size by UI designer's qualitive assessment and scale the UI element to fit it (if it's not big enough). Then if we ask to scale the UI from our standard resolution to whatever, 2x zoom, 4x zoom, 1/2x zoom, etc, we can just take a straightforward multiplication of the pointsize, maybe, to relatively scale with the element?

NOTE: Some languages may have interface elements that are much longer or shorter than English...how can we gracefully handle that?